test/App-test.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
 import { expect } from 'chai';
-import { shallow } from 'enzyme';
+import Enzyme, { shallow } from 'enzyme';
 import sinon from 'sinon';
-
+import Adapter from 'enzyme-adapter-react-16';
 import fetchMock from '../src/fetch-setup';
 import App from '../src/components/App';
 import Filters from '../src/components/Filters';
 import PetBrowser from '../src/components/PetBrowser';
 
+Enzyme.configure({ adapter: new Adapter() });
+
 describe('<App />', () => {
   describe('Filters', () => {
     it('should change filter type', () => {
       const wrapper = shallow(<App />);
-      wrapper.find(Filters).props().onChangeType('micropig');
+      wrapper
+        .find(Filters)
+        .props()
+        .onChangeType('micropig');
```

Comment:
I feel like these test are inflexible because you can't pass onChangeType an event, only an animal. We should allow more flexibility in how they structure this.